SomerenUI/EditDrinkForm.cs

```diff
@@ -13,39 +10,28 @@ public partial class EditDrinkForm : BaseForm
         public EditDrinkForm()
         {
             InitializeComponent();
-            Text = "Create Drink";
-            btnUpdateDrink.Hide();
-            rdoTrue.Checked = true;
+            LoadForm(Properties.Resources.CreateDrink);
         }
 
         public EditDrinkForm(Drink drink)
         {
             InitializeComponent();
-            Text = "Edit Drink";
-            btnCreateDrink.Hide();
             currentDrink = drink;
-            LoadText();
-        }
-
-        private void ShowMessageAndCloseForm(string message, Form form, string arg = null)
-        {
-            ShowMessage(message, arg);
-            form.Close();
+            LoadForm(Properties.Resources.EditDrink);
         }
 
         private void btnCreateDrink_Click(object sender, EventArgs e)
         {
             try
             {
-                ValidateInputs(out double tryGetPrice, out int tryGetStock);
-
-                double price = tryGetPrice;
-                int stock = tryGetStock;
+                string name = ValidateString(txtDrinkName.Text, Properties.Resources.ErrorMessageNoName);
+                double price = ValidatePrice(txtDrinkPrice.Text);
+                int stock = ValidateInt(txtDrinkStock.Text, Properties.Resources.ErrorMessageWrongStock);
                 bool isAlcoholic = rdoTrue.Checked;
 
-                Drink drink = new Drink(price, isAlcoholic, txtDrinkName.Text, stock, 0);
+                Drink drink = new Drink(price, isAlcoholic, name, stock, 0);
```

Comment:
Please, use const or int.Parse(Properties.Resources.Zero) instead of the hardcoded integer.